LiteCore/Storage/DataFile.cc

```diff
@@ -235,10 +251,22 @@ namespace litecore {
                 if (n++ == 0)
                     LogTo(DBLog, "Waiting for %ld other connection(s) to close before deleting %s",
                           otherConnections, shared->path.c_str());
-                if (st.elapsed() > kOtherDBCloseTimeoutSecs)
-                    error::_throw(error::Busy, "Can't delete db file while other connections are open");
-                else
+                if (st.elapsed() > kOtherDBCloseTimeoutSecs) {
+                    ostringstream dbtags;
+                    shared->forOpenDataFiles(nullptr, [&](DataFile* df) {
+                        if (!dbtags.str().empty()) {
+                            dbtags << ", ";
+                        }
+                        dbtags << kDatabaseTags[df->databaseTag()];
+                    });
+                    ostringstream ss;
+                    ss << "Can't delete db file while other connections are open. The open connections are tagged "
+                    << dbtags.str() << ".";
+                    error::_throw(error::Busy, "Can't delete db file while other connections are open."
+                                               " The open connections are tagged %s.", ss.str().c_str());
```

Comment:
This ends up repeating the message twice, since `ss` already contains the entire message.
I would get rid of lines 262-264, then use `dbtags.str().c_str()` in the exception format.

Reply:
Yes, good catch. I meant to have the message, "Can't delete .... The open connections are tagged as x,y.z." But, that gets repeated. Will fix it.